app/routes.js

```diff
@@ -1,9 +1,10 @@
 // const controller = require('./controllers/controller');
 const { healthCheck } = require('./controllers/healthCheck');
+const { getQuote } = require('./controllers/weets');
 
 exports.init = app => {
   app.get('/health', healthCheck);
-  // app.get('/endpoint/get/path', [], controller.methodGET);
+  app.get('/weets', getQuote);
```

Comment:
I think this card is about services only

Reply:
Should I remove the controller and route? Or keep this and be more careful in future work

Reply:
should remove both of them, use the console created by wolox to test it